src/gc/gcee.cpp

```diff
@@ -12,19 +12,16 @@
 
 #ifndef DACCESS_COMPILE
 
-#ifdef ENABLE_PERF_COUNTERS
-PERF_COUNTER_TIMER_PRECISION g_TotalTimeInGC = 0;
-PERF_COUNTER_TIMER_PRECISION g_TotalTimeSinceLastGCEnd = 0;
-#endif
+GCCounterData g_GCCounterData;
```

Comment:
>GCCounterData [](start = 0, length = 13)

you don't need to add this struct - we already have g_GenerationSizes to keep track of the gen sizes; you can just add another field to represent the alloc. although I think we should have 2 alloc numbers, for soh and loh separately, like we used to for perf counters.

Reply:
Thanks - I removed it.